### PR TITLE
#1821 gameplay tips

### PR DESCRIPTION
**Pull request by xseanl**
#1821: Game play tips & scroll bar for the help page added

Longer description:
* As requested in #1821 game play tips for both categories (funny quotes and gameplaytips) are added.
* In the main menu of the game is a "help" link, where all gameplaytips are displayed. After adding tips, they floated below the logbook page. This was fixed by adding a scroll bar.

### CODE REVIEW

**Comment by LinuxDonald** [comment]
For me it look got +1 for merge

@MasterofJOKers what do you say?

**Comment by MasterofJOKers** [comment]
Can you rebase the "typo corrected" Commit into the corresponding commit, where the typo occured? Additionally, I would prefer having only one "added gameplaytips ..." commit. Could you please squash them?

File: horizons/gui/modules/loadingscreen.py [hunk 1]
```diff
@@ -51,7 +51,13 @@
 		"Travel beyond the horizon and discover unknown worlds!",
 		u"War… war never changes",
 		"Support Unknown Horizons with Cookies!",
-		"wow, looks nearly completed \n(Neomex)"
+		"wow, looks nearly completed \n(Neomex)",
+		"Anchor is missing ...",
+		"campfire is lighted",
```
Comment (MasterofJOKers) [inline horizons/gui/modules/loadingscreen.py]: `campfire` starts with a lowercase character while all other quotes seem to start with a uppercase character.

File: horizons/gui/modules/loadingscreen.py [hunk 2]
```diff
@@ -71,7 +77,12 @@
 		_lazy("Build fire stations and doctors to protect your inhabitants from fire and disease."),
 		_lazy("Build storage tents to increase your storage capacity."),
 		_lazy("Make sure every house is in range of a marketplace."),
-		_lazy("Press 'T' to make trees transparent.")
+		_lazy("Press 'T' to make trees transparent."),
+		_lazy("Build storage tents and lookouts to expand your settlement range."),
+		_lazy("To easily see whether the pavilion's range cover all your tents, select it from the build menu and hover it over your existing pavilion. Uncovered tents are shown in yellow. It's a good idea to build a new pavilion in their neighborhood."),
+		_lazy("Make singleplayer more fun with additional computer players by increasing 'AI players' in the new singleplayer menu."),
```
Comment (MasterofJOKers) [inline horizons/gui/modules/loadingscreen.py]: The "new singleplayer menu"? I don't get that.

File: horizons/gui/modules/loadingscreen.py [hunk 3]
```diff
@@ -71,7 +77,12 @@
...
+		_lazy("First steps are easier by looking at how AI players are setting up their settlement."),
+		_lazy("Want funny quotes only? Change the quote types shown here in the settings menu on the game page.")
```
Comment (MasterofJOKers) [inline horizons/gui/modules/loadingscreen.py]: :D

**Comment by MasterofJOKers** [comment]
Only some small remarks and questions. Looks pretty good. Haven't tested it, yet.

**Comment by MasterofJOKers** [comment]
I've got another tip based on a recent conversation on the IRC: 'A marketplace functions for all intents and purposes as a road.' or something like that.

**Comment by xseanl** [comment]
Thanks for your comments. Will work on your recommended changes/questions next week.

**Comment by LinuxDonald** [comment]
@xseanl maybe you could add bookmarks into the help menu as it is in the Settings menu.
The first one could be namend Gameplay Tips an later we could add more Pages to the Help Menu.

**Comment by xseanl** [comment]
Hi, by working my way through the squash command everything resulted in a big mess. Therefore I close this pull request and start a new one.

@LinuxDonald: I decided to not incorporated your idea with the additional bookmarks. I think this is a task better done in a new ticket. Makes testing and searching for errors easier in the future.

**Comment by LinuxDonald** [comment]
Okay im fine with it's

**Pull request closed without merging**